PWGDQ/Tasks/dileptonMuMu.cxx

```diff
@@ -265,25 +265,8 @@ struct DQEventMixing {
       VarManager::ResetValues(0, VarManager::kNVars, fValues);
       VarManager::FillEvent<gkEventFillMap>(event1, fValues);
 
-      auto im1 = slicerMuons.begin();
-      auto im2 = slicerMuons.begin();
-      for (auto& slice : slicerMuons) {
-        if (slice.groupingElement().index() == event1.index()) {
-          im1 = slice;
-          break;
-        }
-      }
-      for (auto& slice : slicerMuons) {
-        if (slice.groupingElement().index() == event2.index()) {
-          im2 = slice;
-          break;
-        }
-      }
-
-      auto muons1 = std::get<soa::Filtered<MyMuonTracksSelected>>(im1.associatedTables());
-      muons1.bindExternalIndices(&events);
-      auto muons2 = std::get<soa::Filtered<MyMuonTracksSelected>>(im2.associatedTables());
-      muons2.bindExternalIndices(&events);
+      auto muons1 = muons.sliceBy(perReducedEvent, event1.globalIndex());
```

Comment:
Suggestion for analyzers: This could be rewritten to use the newer mixing interface and avoid explicit slicing: https://github.com/AliceO2Group/O2Physics/blob/master/Tutorials/src/eventMixing.cxx#L330